.github/workflows/clean_validation_envs.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           MAIN_TRE_ID: ${{ secrets.TRE_ID }}
+          GITHUB_REPOSITORY: ${{ github.repository}}
```

Comment:
Can you try without this line?
Run the cleanup workflow on this branch to confirm

Reply:
Tried, it doesn't work without it